src/renderable/animationsheet.js

```diff
@@ -19,11 +19,12 @@
      * @param {Number} [settings.framewidth] Width of a single frame within the spritesheet
      * @param {Number} [settings.frameheight] Height of a single frame within the spritesheet
```

Comment:
If I draw a Sprite with its anchorPoint at its x and y pos, the debugPanel draws the bounding box incorrectly because it's using this.left (this.pos.x) and this.top (this.pos.y) as the starting point for the bounding rectangle. [debugPanel.js line 200]

Should I override this.left and this.right to take the pixel offset from the anchorPoint into account? I'm not aware of Sprites being used for collision so maybe this isn't much of an issue.